src/fmu/dataio/dataio.py

```diff
@@ -967,10 +967,10 @@ def _generate_aggr_uuid(uuids: list) -> str:
         """Unless aggregation_id; use existing UUIDs to generate a new UUID."""
 
         stringinput = ""
-        for xuuid in uuids:
+        for xuuid in sorted(uuids):
             stringinput += xuuid
 
-        return uuid_from_string(stringinput)
+        return str(uuid_from_string(stringinput))
```

Comment:
Perhaps it would be better if the `uuid_from_string` function do the str() conversion

Reply:
✅